File: src/app/Controllers/DefaultController.ts
import { Request, Response } from 'express';

/**
 * Constroller responsável pela rota padrão do API '/'
 */
class DefaultController {
  /**
   * Método chamado quando a rota '/' é requisitada
   *
   * @param req Request
   * @param res Response
   */
  public index(req:Request, res:Response): Response {
    return res.json({ message: 'Star Wars API' });
  }
}

export default new DefaultController();

File: src/main/kotlin/br/com/swapi/swapi/controllers/DefaultController.kt
package br.com.swapi.swapi.controllers

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

/**
 * Classe que gerencia a rota padrão da API
 */
@RestController
class DefaultController {

    @GetMapping("/")
    fun index(): Map<String, String> {
        return mapOf("message" to "API Star Wars")
    }
}